src/apps/Dashboard/actions/dashboardActions.js

```diff
@@ -43,9 +43,9 @@ export function fetchArticle() {
             for (const i in r) {
               let slug = r[i].slug;
               // add likes, comments and rating of each article in the payload
-              axios.all([fetchArticleLikes(slug), fetchArticleComments(slug), fetchArticleRating(slug)]).then(
+              axios.all([fetchArticleLikes(slug), fetchArticleComments(slug), fetchArticleRating(slug), fetchProfiles()]).then(
                 axios.spread(
-                  (likes, comments, ratings) =>{
+                  (likes, comments, ratings, profiles) =>{
```

Comment:
Why add an argument that you are not using?

Reply:
That was supposed to be done away with. Resolved